### PR TITLE
Use sphinx removed in extension in the documentation

### PR DESCRIPTION
**Pull request by nicoddemus**
Fix https://github.com/pytest-dev/pytest/issues/4568

This is blocked by https://github.com/MrSenko/sphinx-removed-in/pull/1:

---

```
λ tox -re docs
docs create: C:\Users\bruno\pytest\.tox\docs
docs installdeps: -rC:\Users\bruno\pytest/doc/en/requirements.txt
ERROR: invocation failed (exit code 1), logfile: C:\Users\bruno\pytest\.tox\docs\log\docs-1.log
ERROR: actionid: docs
msg: getenv
cmdargs: "'C:\\Users\\bruno\\pytest\\.tox\\docs\\Scripts\\python.EXE' -m pip install '-rC:\\Users\\bruno\\pytest/doc/en/requirements.txt'"

Collecting pygments-pytest>=1.1.0 (from -r C:\Users\bruno\pytest/doc/en/requirements.txt (line 1))
  Using cached https://files.pythonhosted.org/packages/49/3e/fc3a0102c4ae97430d34303c0963cc779e8a1c74a73fe6a04539665fdbb0/pygments_pytest-1.1.0-py2.py3-none-any.whl
Collecting sphinx>=1.8.2 (from -r C:\Users\bruno\pytest/doc/en/requirements.txt (line 2))
  Using cached https://files.pythonhosted.org/packages/ff/d5/3a8727d6f890b1ae45da72a55bf8449e9f2c535a444923b338c3f509f203/Sphinx-1.8.2-py2.py3-none-any.whl
Collecting sphinxcontrib-trio (from -r C:\Users\bruno\pytest/doc/en/requirements.txt (line 3))
  Using cached https://files.pythonhosted.org/packages/09/28/70870892f814a46db677872ba8da59936601d00f1109dc9359b122d3a8fc/sphinxcontrib_trio-1.0.1-py3-none-any.whl
Collecting sphinx-removed-in (from -r C:\Users\bruno\pytest/doc/en/requirements.txt (line 4))
  Using cached https://files.pythonhosted.org/packages/74/04/b6bcf18ddf8ac127b9f0c5dffd20542fdcfbf3db6b689947217bd46f4980/sphinx-removed-in-0.1.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\bruno\AppData\Local\Temp\pip-install-sqas3ip2\sphinx-removed-in\setup.py", line 5, in <module>
        __version__ = __import__('sphinx_removed_in').__version__
      File "C:\Users\bruno\AppData\Local\Temp\pip-install-sqas3ip2\sphinx-removed-in\sphinx_removed_in\__init__.py", line 1, in <module>
        from sphinx.locale import versionlabels
    ModuleNotFoundError: No module named 'sphinx'
```

### CODE REVIEW

**Comment by nicoddemus** [comment]
Hmmm same error even though https://github.com/MrSenko/sphinx-removed-in/pull/1 has been fixed and we are using `0.1.1`. I also checked the PyPI package and it contains the correct `setup_requires` keyword parameter... any ideas @RonnyPfannschmidt?

**Comment by RonnyPfannschmidt** [comment]
@nicoddemus setup_requires happens too late to work out sanely there, aka its evaluated after that import happens

**Comment by nicoddemus** [comment]
Duh of course. 

I suppose the simplest fix would be to stop importing `sphinx` at the top-level of `sphinx_removed_in`.

cc @MrSenko

**Comment by nicoddemus** [comment]
https://github.com/MrSenko/sphinx-removed-in/issues/3

File: doc/en/deprecations.rst [hunk 1]
```diff
@@ -423,10 +423,28 @@ Users should just ``import pytest`` and access those objects using the ``pytest`
 
 This has been documented as deprecated for years, but only now we are actually emitting deprecation warnings.
 
+``Node.get_marker``
```
Comment (nicoddemus) [inline doc/en/deprecations.rst]: Moved those two deprecations to near the other ones from 4.0.

File: doc/en/requirements.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
 pygments-pytest>=1.1.0
 sphinx>=1.8.2
 sphinxcontrib-trio
+sphinx-removed-in>=0.1.3
```
Comment (nicoddemus) [inline doc/en/requirements.txt]: @atodorov thanks again! 👍

**Comment by nicoddemus** [comment]
Ready for review. 👍

**Comment by Tadaboody** [comment]
Is there a way to see the compiled docs for the review?

**Comment by RonnyPfannschmidt** [comment]
@nicoddemus can we push this branch to the main repo so we can trigger a rtd build?

**Comment by nicoddemus** [comment]
Hmm I did a few hours ago and I still don't see it listed under the "versions" option. Weird. 🤔

**Comment by nicoddemus** [comment]
Ahh there you go: https://docs.pytest.org/en/sphinx-removed-in/deprecations.html

**Comment by nicoddemus** [comment]
Targetting `master` now. 👍

**Pull request merged**